docs/install/README.md

```diff
@@ -0,0 +1,422 @@
+
+Create a kustomization directory that builds from the `main` branch:
+
+```sh
+K8SSANDRA_OPERATOR_HOME=$(mktemp -d)
+cat <<EOF >$K8SSANDRA_OPERATOR_HOME/kustomization.yaml
+resources:
+- github.com/k8ssandra/k8ssandra-operator/config/default?ref=main
+
+images:
+- name: k8ssandra/k8ssandra-operator
+  newTag: 7a2d65bb
+EOF
+```
+Now install the operator:
+
+```
+kustomize build $K8SSANDRA_OPERATOR_HOME | kubectl apply -f -
+```
+This installs the operator in the `default` namespace.
+
+If you just want to generate the maninfests then run:
+
+```
+kustomize build $K8SSANDRA_OPERATOR_HOME
+```
+
+Verify that the following CRDs are installed:
+
+* `cassandradatacenters.cassandra.datastax.com`
+* `certificaterequests.cert-manager.io`
+* `certificates.cert-manager.io`
+* `challenges.acme.cert-manager.io`
+* `clientconfigs.k8ssandra.io`
+* `clusterissuers.cert-manager.io`
+* `issuers.cert-manager.io`
+* `k8ssandraclusters.k8ssandra.io`
+* `orders.acme.cert-manager.io`
+* `replicatedsecrets.k8ssandra.io`
+* `stargates.k8ssandra.io`
+
+
+Check that there are two Deployments. The output should look similar to this:
+
+```
+kubectl get deployment
+NAME                 READY   UP-TO-DATE   AVAILABLE   AGE
+cass-operator        1/1     1            1           2m
+k8ssandra-operator   1/1     1            1           2m
+```
+### Install into a different namespace
+This slight variation demonstrates how to install the operator into a different namespace.
+
+Create a namespace named `k8ssandra-operator`:
+
+```sh
+kubectl create ns k8ssandra-operator
+```
+Create a kustomization directory:
+
+```sh
+K8SSANDRA_OPERATOR_HOME=$(mktemp -d)
+cat <<EOF >$K8SSANDRA_OPERATOR_HOME/kustomization.yaml
+namespace: k8ssandra-operator
+
+resources:
+- github.com/k8ssandra/k8ssandra-operator/config/default?ref=main
+
+images:
+- name: k8ssandra/k8ssandra-operator
+  newTag: 7a2d65bb
+EOF
+```
+
+Note that the `namespace` property has been added. This property tells Kustomize to apply a transformation on all resources that specify a namespace.
+
+Now install the operator:
+
+```
+kustomize build $K8SSANDRA_OPERATOR_HOME | kubectl apply -f -
+```
+This installs the operator in `k8ssandra-operator`.
+
+## Deploy a K8ssandraCluster
+Now we will deploy a K8ssandraCluster that consists of a 3-node Cassandra cluster and a Stargate node.
+
+```sh
+cat <<EOF | kubectl apply -f -
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: demo
+spec:
+  cassandra:
+    cluster: demo
+    serverVersion: "4.0.0"
+    datacenters:
+      - metadata:
+          name: dc1
+        size: 3
+        storageConfig:
+          cassandraDataVolumeClaimSpec:
+            storageClassName: standard
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: 5Gi
+        config:
+          jvmOptions:
+            heapSize: 512M
+        stargate:
+          size: 1
+          heapSize: 256M
+EOF 
```

Comment:
Confirming that these instructions bring up a 3 node Cassandra cluster. I still get an error message on the stargate node 
```
0/4 nodes are available: 1 node(s) had taint {node-role.kubernetes.io/master: }, that the pod didn't tolerate, 3 node(s) didn't match pod affinity/anti-affinity, 3 node(s) didn't match pod anti-affinity rules.
```

Why are we tainting k8s masters?

Reply:
We aren't tainting master. That is done by k8s itself. Please provide the exact command line you ran.

Reply:
Exactly as per the instructions - 

```
./setup-kind-multicluster.sh --kind-worker-nodes 4
kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml

K8SSANDRA_OPERATOR_HOME=$(mktemp -d)
cat <<EOF >$K8SSANDRA_OPERATOR_HOME/kustomization.yaml
resources:
- github.com/k8ssandra/k8ssandra-operator/config/default?ref=main

images:
- name: k8ssandra/k8ssandra-operator
  newTag: 7a2d65bb
EOF

kustomize build $K8SSANDRA_OPERATOR_HOME | kubectl apply -f -
```

Perhaps `scripts/setup-kind-multicluster.sh` counts the master as a worker node and therefore under-provisions the cluster? Or maybe my `getopts` is interfering?